### ISSUE TITLE
rtd ist not following language code update for Chinese

### ISSUE THREAD

**Issue by Flohack74** [rtfd/readthedocs.org#4411]
## Details

* Try to configure Chinese traditional and simplified languages

## Expected Result

* The selection should be zh_Hans and zh_Hant

## Actual Result

* Selection is zh_CN and zh_TW

See https://code.djangoproject.com/ticket/18419 for example. zh_CN is no longer common sense, and we got issues with linking our Weblate to rtd, since Weblate does not support zh_CN anymore.

**Comment by RichardLitt** [comment]
@Flohack74 Can this be closed, now?

**Comment by humitos** [comment]
I suppose we should check some things here:

* our current translations under transifex
* this change is supported by Sphinx
* old URLs that point to `zn_CN`

**Comment by agjohnson** [comment]
Sounds like we need to update our language selector to add the new identifiers for chinese lanuages? We can leave the old identifiers probably? Optionally, we can hide the deprecated identifier for new projects.

Sphinx should be agnostic to this.

I imagine a PR should be fairly quick to put together for this.

**Comment by Flohack74** [comment]
Yeah I think thats the best way, stay compatible with whats there, but try to offer only the new shortcuts for new stuff.

**Comment by ericholscher** [comment]
No activity here for 7 years, so I'm just going to close this. I believe it was fixed via migrations 0107_alter_project_language.py (zh_CN→zh-cn, zh_TW→zh-tw) and 0124_remove_zh_locale.py.

**Comment by akien-mga** [comment]
How does one migrate from using `zh_CN`/`zh_TW` to `zh_Hans`/`zh_Hant` on a live RTD instance?

We've tried this just a couple of months ago for the Godot documentation and we had to revert back (https://github.com/godotengine/godot-docs-l10n/commit/e62310a76497964419a82e7bb5252b4fd2d4cb34).
